packages/react-i18n/src/i18n.ts

```diff
@@ -525,6 +527,29 @@ export class I18n {
       return this.translate('date.humanize.tomorrow', {time});
     }
 
+    if (isLessThanOneWeekAway(date)) {
+      const weekday = this.formatDate(date, {
+        ...options,
+        weekday: 'long',
+      });
+      return this.translate('date.humanize.lessThanOneWeekAgo', {
```

Comment:
I re-used the `date.humanize.lessThanOneWeekAgo` key for translations here and for the `isLessThanOneYearAway` case below since the format is the same, but I don't know if that's the right thing to do since the naming could lead to confusion.